package.json

```diff
@@ -7,7 +7,11 @@
     "node": "^16.3.0"
   },
   "scripts": {
-    "start": "node bin/www",
+    "clean": "rm -rf dist",
+    "transpile": "babel src -d dist --copy-files",
+    "build": "npm run clean && npm run transpile",
+    "serve": "npm run build && node dist/bin/www.js",
+    "start": "nodemon --exec babel-node src/bin/www.js",
```

Comment:
Note:
For prod: npm run serve
For dev: npm run start

babel-node is not appropriate for production: https://babeljs.io/docs/en/babel-node.html